Documentation/services/pcn-firewall/firewall.rst

```diff
@@ -35,15 +35,23 @@ How to use
 Ingress ad egress chains
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-The service supports independent ingress and egress policy chains, with two different policy sets.
+The service supports independent _ingress_ (packets that are try to reach your TCP/IP stack) and _egress_ (pac policy chains, with two different policy sets:
+  - **ingress**: packets that come from the external world and that are try to reach the _inside_ of your network;
+  - **egress**: packets that come from your inside network and that are trying to reach the external world.
```

Comment:
If I'm not wrong ` must be used on ReadTheDocs to create italic text instead of \_
```suggestion
The service supports independent `ingress` and `egress` policy chains, with two different policy sets:
  - **ingress**: packets that come from the external world and that are trying to reach the `inside` of your network;
  - **egress**: packets that come from your inside network and that are trying to reach the external world.
```

Reply:
Thanks for checking my the typos.
With respect to italic, ReadTheDocs supports "_" as a marked for italic, so I'm going to reject your suggestions in that respect. Thanks!